torch/_dynamo/utils.py

```diff
@@ -139,12 +141,10 @@
 # profiling compilation time by function
 compilation_time_metrics: Dict[str, List[float]] = {}
 
-# profiling compilation time by frame phase
-frame_phase_timing: Dict[str, Dict[str, float]] = collections.defaultdict(
-    lambda: collections.defaultdict(float)
-)
-
-codecache_metrics: Counter[str] = collections.Counter()
+# This supports calculate_time_spent(), which reports cumulative times (in seconds)
+# across the process for any "phase" populated by dynamo_timed. Reset if
+# reset_frame_count() is called.
```

Comment:
This is seconds because it's the grandfathered second-based APIs, is that right?

Reply:
Ugh, no I named it wrong.